todo_app/templates/index.html

```diff
@@ -1,18 +1,104 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
+{% block title %} [to_do || !to_do] app {% endblock %}
 
 {% block content %}
+<style type="text/css">
```

Comment:
Writing good HTML/css isn't really part of the course, but some minor points:

- You normally want all your styling to be in a css file that you load rather than including in the html file.
- The class names could be more descriptive than `display-1` etc.

Reply:
So these three points relating to the style are linked. I was struggling to get an external css file to work, but removed the dead code and elected for a style section instead. I left the unused file in the repo so it was there to come back to - I was just trying a very obvious style change to see if it worked..

Your note about the body style just got me thinking though. Is the reason that it couldn't apply the css external file because I was trying to do it in the index.html file instead of the layout.html file? 

Fair point on the class names, I wasn't sure the best way to preserve the other attributes of the pre-made "display" classes while making the tweaks I needed. Is it possible to inherit these properties, or to do a deep copy of the class before changing it?